Sources/Testing/Testing.docc/exit-testing.md

```diff
@@ -126,10 +127,10 @@ value using the `as` operator:
 ```
 
 Every value you capture in an exit test must conform to [`Sendable`](https://developer.apple.com/documentation/swift/sendable)
-and [`Codable`](https://developer.apple.com/documentation/swift/codable). Each
-value is encoded by the parent process using [`encode(to:)`](https://developer.apple.com/documentation/swift/encodable/encode(to:))
-and is decoded by the child process [`init(from:)`](https://developer.apple.com/documentation/swift/decodable/init(from:))
-before being passed to the exit test body.
+and [`Codable`](https://developer.apple.com/documentation/swift/codable). The
+testing library encodes each value using [`encode(to:)`](https://developer.apple.com/documentation/swift/encodable/encode(to:))
+in the parent process, and decodes it using [`init(from:)`](https://developer.apple.com/documentation/swift/decodable/init(from:))
+in the child process, before passing it to the exit test body.
```

Comment:
The new commas are incorrect, at least the way I was taught (circa 1854.)

Reply:
The words "and", and "before", are acting as coordinating conjunctions between independent clauses in this sentence. While you can always find a grammar "rule" that says you should use a comma in the same place that another "rule" says you shouldn't use one, I think the commas separate the independent clauses here.

Reply:
We could split the difference and make this a list of steps instead?